packages/cfengine/build.sh

```diff
@@ -0,0 +1,27 @@
+TERMUX_PKG_HOMEPAGE=https://cfengine.com/
+TERMUX_PKG_DESCRIPTION="CFEngine is a configuration management technology that helps systems administrators automate and ensure the availability, security and compliance of mission-critical applications and services."
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_VERSION=(3.15.0-2
+                    3.15.0-2)
+TERMUX_PKG_SRCURL=(https://cfengine-package-repos.s3.amazonaws.com/tarballs/cfengine-3.15.0.tar.gz
+                   https://github.com/cfengine/masterfiles/archive/3.15.0-2-build1.tar.gz)
+                   #https://cfengine-package-repos.s3.amazonaws.com/enterprise/Enterprise-3.15.0/misc/cfengine-masterfiles-3.15.0.pkg.tar.gz)
+TERMUX_PKG_SHA256=(fa53e137f850eb268a8e7ae4578b5db5dc383656341f5053dc1a353ed0288265
+                   d47311079503f4e56a2d5de4c84d16e21a067093fbb0b1290aac6cdf1dd9a228)
+                   #ffeb1dad5b14615f0b4cd7032d07982708484cb589f6c69ee075d122c5babdca)
+TERMUX_PKG_DEPENDS="liblmdb, openssl, libandroid-glob, pcre"
+#TERMUX_PKG_BUILD_DEPENDS="liblmdb, openssl, libandroid-glob, pcre"
```

Comment:
Remove extraneous comments. The above are runtime dependencies that will be installed in users' Termux apps, while the commented out are dependencies you only need when building. Split the dependencies up according to where they're needed.